test/pauseTokenTests.js

```diff
@@ -15,17 +15,16 @@ require('chai')
 
 const expect = require('chai').expect
 
-contract('polyTokenPause', async function([miner, owner, investor, wallet]) {
+contract('polyTokenPause', async function([miner, owner, investor, investor2, wallet, presale_wallet]) {
```

Comment:
investor2? maybe we could call this something else to make it more tidy. any suggestions?

Reply:
contributor?

Reply:
👀